receiver/prometheusreceiver/internal/transaction_test.go

```diff
@@ -197,14 +270,42 @@ func nopObsRecv() *obsreport.Receiver {
 	})
 }
 
+func createTestExemplar(value float64, ts int64, ls labels.Labels) exemplar.Exemplar {
+	return exemplar.Exemplar{
+		Value:  value,
+		Ts:     ts,
+		Labels: ls,
+	}
+}
+
+func createTestExemplarWithoutTimestamp(value float64, ls labels.Labels) exemplar.Exemplar {
+	return exemplar.Exemplar{
+		Value:  value,
+		Labels: ls,
+	}
+}
+
+func createTestExemplarWithoutValue(ts int64, ls labels.Labels) exemplar.Exemplar {
+	return exemplar.Exemplar{
+		Ts:     ts,
+		Labels: ls,
+	}
+}
+
 func TestMetricBuilderCounters(t *testing.T) {
 	tests := []buildTestData{
 		{
 			name: "single-item",
 			inputs: []*testScrapedPage{
 				{
 					pts: []*testDataPoint{
-						createDataPoint("counter_test", 100, "foo", "bar"),
+						createDataPoint(
+							"counter_test",
+							100,
+							[]exemplar.Exemplar{
+								createTestExemplar(1, 1663113420863, []labels.Label{{model.MetricNameLabel, "counter_test"}, {model.JobLabel, "job"}, {model.InstanceLabel, "instance"}, {"foo", "bar"}}),
+							},
+							"foo", "bar"),
```

Comment:
Make two tests: `single-item` and `singel-item-with-exemplars`. Then can remove the exemplars from the other places like the `two-items`, same for all types.

So we should have only one test for the builders `one-item-with-exemplar`. I believe that it enough, considering that we also have the previous tests (for errors, inexistent metrics, etc.).

Reply:
Done